Move to @obeonetwork/react-trello new version to avoid dependencies warning

* [ ] **I have checked that this bug has not yet been reported by someone else**
* [ ] **I have checked that this bug appears on Chrome**
* [ ] **I have specified the version** :
- latest
- other: please specify
* [ ] **I have specified my environment** : 
- All
- my computer: Windows? Linux? Cloud?
- Cloud

We have warning because of react-trello which depends on too old react-redux version: 

```
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: react-redux@5.1.2
npm WARN Found: react@17.0.2
npm WARN node_modules/react
npm WARN   react@"17.0.2" from the root project
npm WARN   49 more (@apollo/client, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react@"^0.14.0 || ^15.0.0-0 || ^16.0.0-0" from react-redux@5.1.2
npm WARN node_modules/@ObeoNetwork/react-trello/node_modules/react-redux
npm WARN   react-redux@"^5.0.7" from @ObeoNetwork/react-trello@2.4.1
npm WARN   node_modules/@ObeoNetwork/react-trello
npm WARN 
npm WARN Conflicting peer dependency: react@16.14.0
npm WARN node_modules/react
npm WARN   peer react@"^0.14.0 || ^15.0.0-0 || ^16.0.0-0" from react-redux@5.1.2
npm WARN   node_modules/@ObeoNetwork/react-trello/node_modules/react-redux
npm WARN     react-redux@"^5.0.7" from @ObeoNetwork/react-trello@2.4.1
npm WARN     node_modules/@ObeoNetwork/react-trello
```

The react-redux version has been upgrade in the react-trello 2.4.11 release

--- a/CHANGELOG.adoc
+++ b/CHANGELOG.adoc
@@ -54,6 +54,7 @@ It's current location is only there to simplify the transition to the new Sirius
 === Dependency update
 
 - https://github.com/eclipse-sirius/sirius-web/issues/2783[#2783] [diagram] Remove our dependency to Sprotty and Snabbdom
+- https://github.com/eclipse-sirius/sirius-web/issues/3105[#3105] [deck] Move to @obeonetwork/react-trello 2.4.11 to avoid warnings because of react-redux depending on react 16.
 
 === Bug fixes
 

--- a/packages/deck/frontend/sirius-components-deck/package.json
+++ b/packages/deck/frontend/sirius-components-deck/package.json
@@ -32,7 +32,7 @@
   "peerDependencies": {
     "@apollo/client": "3.8.1",
     "@eclipse-sirius/sirius-components-core": "*",
-    "@ObeoNetwork/react-trello": "2.4.10",
+    "@ObeoNetwork/react-trello": "2.4.11",
     "@material-ui/core": "4.12.4",
     "@material-ui/icons": "4.11.3",
     "@xstate/react": "1.6.3",
@@ -49,7 +49,7 @@
     "@material-ui/icons": "4.11.3",
     "@types/react": "17.0.37",
     "@vitejs/plugin-react": "4.0.4",
-    "@ObeoNetwork/react-trello": "2.4.10",
+    "@ObeoNetwork/react-trello": "2.4.11",
     "@xstate/react": "1.6.3",
     "graphql": "16.8.0",
     "prettier": "2.7.1",

--- a/packages/sirius-web/frontend/sirius-web/package.json
+++ b/packages/sirius-web/frontend/sirius-web/package.json
@@ -30,7 +30,7 @@
     "@material-ui/icons": "4.11.3",
     "@material-ui/lab": "4.0.0-alpha.61",
     "@ObeoNetwork/gantt-task-react": "0.3.13",
-    "@ObeoNetwork/react-trello": "2.4.10",
+    "@ObeoNetwork/react-trello": "2.4.11",
     "@types/react": "17.0.37",
     "@types/react-router-dom": "5.3.3",
     "@xstate/react": "1.6.3",

